### PR TITLE
pyEWS: add integration + playbook test

### PR DESCRIPTION
**Pull request by erezh31**

### CODE REVIEW

**Comment by aviadl** [comment]
@erezh31 can you please paste here the test playbook ?

**Comment by erezh31** [comment]
@aviadl https://demo2.demisto.com/#/WorkPlan/39260

**Comment by meirwah** [comment]
@Spivaka @aviadl can you add if something is missing from EWS commands here?

**Comment by aviadl** [comment]
I reviewed with @erezh31 and mapped the missing parts
Command wise we are fine

File: Tests/schemas/integration.yml [hunk 1]
```diff
@@ -56,6 +56,8 @@ mapping:
         type: str
       isfetch:
         type: bool
+      runonce:
+        type: bool
```
[the remark this answers is not in the document]
Comment (erezh31) [reply Tests/schemas/integration.yml]: It was for the scheme test to pass... It's new field we added recently I think

[the remark this answers is not in the document]
Comment (meirwah) [reply Tests/schemas/integration.yml]: then update from master plz

**Comment by erezh31** [comment]
missing parts discussed with @aviadl already fixed.
new playbook test:
https://demo2.demisto.com/#/WorkPlan/39320

File: Integrations/integration-pyEWS.yml [hunk 2]
```diff
@@ -0,0 +1,1137 @@
...
+        items = map(lambda x: Item(item_id=x), item_ids)
+        result = list(account.fetch(ids=items))
+        result = [x for x in result if not isinstance(x, ErrorItemNotFound)]
+        for item in result:
+            item.folder = Folder(account=account)
+        return result
+
+
+    def get_item_from_mailbox(account, item_id):
+        return get_items_from_mailbox(account, [item_id])[0]
```
Comment (meirwah) [inline Integrations/integration-pyEWS.yml]: maybe check here for empty array?

File: Tests/schemas/integration.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (erezh31) [reply Tests/schemas/integration.yml]: it's not in master...

File: Integrations/integration-pyEWS.yml [hunk 3]
```diff
@@ -0,0 +1,1137 @@
...
+        return last_run
+
+
+    def fetch_last_emails(account, folder_name='Inbox', since_datetime=None, exclude_ids=None):
+        qs = get_folder_by_path(account.root, folder_name)
+        if since_datetime:
+            qs = qs.filter(datetime_received__gte=since_datetime)
+        else:
+            if not FETCH_ALL_HISTORY:
+                last_10_min = EWSDateTime.now(tz=EWSTimeZone.timezone('UTC')) - timedelta(minutes=10)
```
Comment (meirwah) [inline Integrations/integration-pyEWS.yml]: what EWS is in a different timezone from Demisto server?
Do you support such configuration here?

Comment (erezh31) [reply Integrations/integration-pyEWS.yml]: demisto timezone does not matter. EWSDateTime is always with timezone. the lib take care of it.

**Comment by erezh31** [comment]
circle failed on other unstable tests
@aviadl plz review the changes

File: Integrations/integration-pyEWS.yml (hunk 2 above)
Comment (erezh31) [reply Integrations/integration-pyEWS.yml]: fixed

**Comment by meirwah** [comment]
@erezh31 any reason why not merging this?

**Comment by erezh31** [comment]
No.
@aviadl shuold approve (CI test passed... other test has failed)

**Comment by erezh31** [comment]
fixes: https://github.com/demisto/etc/issues/10784

**Pull request merged**